### ISSUE TITLE
`missingReturn` is not recognizable for `@suppress` in closure mode

### ISSUE THREAD

**Issue by tykus160** [gajus/eslint-plugin-jsdoc#1349]
## Expected behavior

No error when `missingReturn` is on `@suppress` annotation, as it is a valid value according to [Closure wiki](https://github.com/google/closure-compiler/wiki/@suppress-annotations).

## Actual behavior

ESLint throws following error:
```
Syntax error in suppress type: missingReturn 
```

## ESLint Config

```js
{
  settings: {
    jsdoc: {
      mode: 'closure',
    },
  },
  rules: {
    'jsdoc/valid-types': 'error',
  },
}
```

## ESLint sample

```js
/** @suppress {missingReturn} */
```

## Environment

- Node version: 22.11.0
- ESLint version 9.18.0
- `eslint-plugin-jsdoc` version: 50.6.1
